### DISCUSSION TITLE
Problem with line-wise selection with triple left-click

### DISCUSSION THREAD

**Discussion by monyxie** [vscode-neovim/vscode-neovim#2303] [Q&A] [answered]
Hi, all.

I noticed when I use triple left-click to do line-wise selection in VSCode, it also selects the first character on the next line. This behavior deviates from Neovim, in which only the current line is selected. It's been bothering me for a while now.
Is it the expected behavior? How can I make it behave like Neovim? Any help is appreciated.

**Comment by justinmk** [comment, marked as answer]
Sounds like a off-by-one bug. Please create an issue.

**Reply by monyxie** [reply]
Created. Thanks.
